app/models/partner.rb

```diff
@@ -50,4 +50,13 @@ def csv_export_attributes
   def register_on_partnerbase
     UpdateDiaperPartnerJob.perform_async(id)
   end
+
+  def quantity_year_to_date
+    distributions
+      .includes(items: :line_items)
+      .where("line_items.created_at > ?", Time.zone.today.beginning_of_year)
+      .references(items: :line_items).map do |distribution|
+      distribution.items.map(&:line_items)
+    end.flatten.sum(&:quantity)
```

Comment:
:100: :tada: :heart: :heart_eyes: 

that's some beautiful querying right there